content/members/apapadopoulou.md

```diff
@@ -1,5 +1,5 @@
 title: Angeliki - Papadopolou date: 20200528 joined_date: 20200529 category: members member_category: Researchers id: apapadopoulou
-givenname: Angeliki surname: Papadopoulou email: angelikeepap@gmail.com github: apapadopoulou linkedin: aggeliki-papadopoulou-643a4116a photo: 
+givenname: Angeliki surname: Papadopoulou email: angelikeepap@gmail.com github: apapadopoulou linkedin: aggeliki-papadopoulou-643a4116a photo: apapadopoulou.JPG
```

Comment:
Should be 
```
givenname: Angeliki
surname: Papadopoulou
email: angelikeepap@gmail.com
github: apapadopoulou
linkedin: aggeliki-papadopoulou-643a4116a
photo: apapadopoulou.JPG
```
rather than
```
givenname: Angeliki surname: Papadopoulou email: angelikeepap@gmail.com github: apapadopoulou linkedin: aggeliki-papadopoulou-643a4116a photo: apapadopoulou.JPG
```